openml/flows/flow.py

```diff
@@ -131,8 +131,10 @@ def __init__(self, name, description, model, components, parameters,
         self.language = language
         self.dependencies = dependencies
         self.flow_id = flow_id
-
-        self._extension = get_extension_by_flow(self)
+        if extension is None:
```

Comment:
Do you want me to remove this? Would work without it but I found it more explicit to directly give the extension.

Reply:
This is totally fine by me.